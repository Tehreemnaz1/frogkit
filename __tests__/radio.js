import { mount } from 'vue-test-utils'
import Vue from 'vue'
import Radio from '@/components/radio'

describe('Radio button component', () => {

  it('should render component and match snapshot', () => {
    const wrapper = mount(Radio)
    expect(wrapper.html()).toMatchSnapshot()
  })

  it('should render disabled radio button', () => {
    const wrapper = mount(Radio, { propsData: { disabled: true } })
    expect(wrapper.html()).toMatchSnapshot()
  })

  it('should render checked radio button', () => {
    const wrapper = mount(Radio, { propsData: { checked: true } })
    expect(wrapper.html()).toMatchSnapshot()
  })

})
