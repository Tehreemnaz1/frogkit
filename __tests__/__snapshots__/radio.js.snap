// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Radio button component should render checked radio button 1`] = `<label class="fk-radio"><input type="radio"> </label>`;

exports[`Radio button component should render component and match snapshot 1`] = `<label class="fk-radio"><input type="radio"> </label>`;

exports[`Radio button component should render disabled radio button 1`] = `<label class="fk-radio fk-radio--disabled"><input type="radio" disabled="disabled"> </label>`;
